specification/azuredata/resource-manager/Microsoft.AzureData/preview/2019-07-24-preview/azuredata.json

```diff
@@ -1094,22 +1633,12 @@
             "$ref": "#/parameters/resourceGroupName"
           },
           {
-            "name": "postgresInstanceName",
-            "description": "Name of Postgres Instance",
+            "name": "dataControllerName",
             "in": "path",
             "required": true,
             "type": "string",
             "x-ms-parameter-location": "method"
           },
-          {
```

Comment:
Removing required parameter is breaking change.

Reply:
Hey @raych1 , the parameter was not removed. It's just the way GitHub shows the diff. Thanks.